src/elements/Image/Image.tsx

```diff
@@ -20,13 +20,14 @@ function getContainerStyle(node: ImageNode): CSSProperties {
         return {};
     }
 
-    const width = `${parseFloat(node.width).toFixed(2)}%`;
+    const maxWidth = `${node.file.original_width}px`;
+    const unit = node.width.slice(-1) === '%' ? '%' : 'px';
 
-    if (width === `${(100).toFixed(2)}%`) {
-        return {};
+    if (unit === '%' && parseFloat(node.width).toFixed(2) === Number(100).toFixed(2)) {
```

Comment:
I also made sure that the renderer supports both `%` and `px` values in the `width` attribute.